conftest.py

```diff
@@ -0,0 +1,132 @@
+from django.contrib.auth import get_user_model
+from player.models import BallGame, Player
+from player_rating.models import PlayerRating, Rating
+from court.models import Court
+from game_event.models import GameEvent
+from decimal import Decimal
+from django.utils import timezone
+from message.models import Message
+from game_event_player.models import GameEventPlayer
+from court_ball_game.models import CourtBallGame
+import pytest
+
+
+@pytest.fixture
+def user():
+    return get_user_model().objects.create_user(
+        username='testuser',
+        password='testpass')
+
+
+@pytest.fixture
+def user2():
+    return get_user_model().objects.create_user(
+        username='testuser2',
+        password='testpass')
+
+
+@pytest.fixture
+def user3():
+    return get_user_model().objects.create_user(
+        username='testuser3',
+        password='testpass')
+
+
+@pytest.fixture
+def player(user):
+    player = Player.objects.create(
+        user=user,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer)
+    return player
+
+
+@pytest.fixture
+def player2(user2):
+    player = Player.objects.create(
+        user=user2,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer)
+    return player
+
+
+@pytest.fixture
+def player3(user3):
+    player = Player.objects.create(
+        user=user3,
+        birth_date='1990-01-01',
+        favorite_ball_game=BallGame.Soccer)
+    return player
+
+
+@pytest.fixture
+def players(player, player2, player3):
+    return tuple([player, player2, player3])
+
+
+@pytest.fixture
+def player_rating(player):
+    player_rating_fixture = PlayerRating.objects.create(
+        ball_game=BallGame.Soccer,
+        user_id=player,
+        rating=Rating.Five)
+    return player_rating_fixture
+
+
+@pytest.fixture
+def court():
+    court = Court.objects.create(
+        x=Decimal('11'),
+        y=Decimal('22'),
+        city='TEST_CITY',
+        neighborhood='TEST_NEIGHBORHOOD',
+        max_players=30)
+    return court
+
+
+@pytest.fixture
+def game_event(court):
+    return GameEvent.objects.create(
+        id=2,
+        time=timezone.now(),
+        level_of_game=3,
+        min_number_of_players=2,
+        max_number_of_players=5,
+        court_id=court,
```

Comment:
At @MaayanMashhadi's PR #55 the filed `court_id `changed to just court. so I don't know which PR will be merged first but I'd change anywhere `court_id` appears to court as @MaayanMashhadi expected. 
from a quick overview I see it in `def game_event(court):` and `def game_event2(court)`:

Reply:
> At @MaayanMashhadi's PR #55 the filed `court_id `changed to just court. so I don't know which PR will be merged first but I'd change anywhere `court_id` appears to court as @MaayanMashhadi expected. from a quick overview I see it in `def game_event(court):` and `def game_event2(court)`:

Thank you Idan you're right.
I need to do same changes in GameEventPlayer entity (see issue #56 )

Reply:
You are correct, I would've change it immediately, but I'm afraid unit test will fail, unless I make sure all instances will be court at every location.
problem is that it would render Maayan's PR redundant.

Don't you agree?

I hope Maayan's PR will be merged first then I'll do quick rebase.